rfc/2023/2023-05-01-va-rfc-frontend-performance.md

```diff
@@ -0,0 +1,156 @@
+# RFC: VFS teams shall monitor performance and gradually adopt performance budgets
+
+<!--
+The title is what you want comments on. Use the active voice in a future tense.
+Example:
+    - The website will be built using the XZY framework
+-->
+
+* Comment Deadline: `2023-06-10`
+* Team Crew and Name: <!--e.g. Platform/AMT-->
+* Authors:
+  * [@dcloud](https://github.com/dcloud)
+  * [Contributing Author 1 Github Handle](https://github.com/ContributingAuthor1Handle)
+  * [Contributing Author 2 Github Handle](https://github.com/ContributingAuthor2Handle)
+* [Original RFC Pull Request](https://github.com/department-of-veterans-affairs/va.gov-platform-arch/pull/-1)
+
+## Background
+
+> Explain the current state. What is the problem? What needs to happen? Provide enough background for someone new to the problem space to understand this decision. Use active voice, present tense, and decisive language.
+> Example: We need to choose how the website will render in the browser. There are several competing frameworks to choose from.
+
+Currently, VA.gov frontend pages perform poorly for a significant number of Veterans. In late 2022, 48.35% of users to va.gov were on relatively slow mobile devices (Source: Google Analytics, 7/26/22-8/01/22). A significant amount are also on slow third-party devices, such as library desktops, per [government research](https://docs.fcc.gov/public/attachments/DOC-357270A1.pdf). Automated reports on the [Frontend support dashboard](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/frontend-support-dashboard/lighthouse-performance-report/) and in Datadog via [RUM's Core Web Vitals](https://docs.datadoghq.com/real_user_monitoring/browser/monitoring_page_performance/#core-web-vitals) show us that many pages on VA.gov perform poorly for Veterans. We need to optimize VA.gov frontend applications for better performance.
+
+Performance affects people's experience with websites, and poor performance can prevent people from using a website. For Veterans, this can affect whether they can access the services they need online. However, [industry research shows](https://infrequently.org/2022/12/performance-baseline-2023/) how we can budget for better, more equitable performance. We should follow this guidance and adopt practices that help us improve how our web applications perform for all Veterans.
```

Comment:
Might be a good section to reference "accessibility beyond compliance" and the adhoc a11y playbook

Reply:
Updated. Let me know what you think. Direct edits/contributions also welcome!